src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx

```diff
@@ -491,9 +491,11 @@ export const TakeProfitAmountUp = () => {
                             <FormulaValue>
                                 <Localize translate_text="((3400" />
                                 <PointerContainer width="200px" ml="-75px" height="89px">
-                                    <PointerDot />
-                                    <PointerStick height="62px" />
-                                    <PointerText>
+                                    <PointerDot ml="100px" />
+                                    <PointerStick ml="100px" height="31px" />
+                                    <PointerStickHorizontal w="125px" ml="-25px" />
+                                    <PointerStick ml="-150px" height="31px" />
+                                    <PointerText ml="-150px">
```

Comment:
I think, unfortunately, that this component is not very sustainable since we use hard coded pixel values. This makes the component not flexible for changing sentence sizes depending on the language. I think comparing english and italian languages is a good way to test because it will show you how things will break due to the changing sentence sizes. Best is to find a better way to rebuild this component.

Reply:
Yeah, it's true @hubert-deriv. Let me try to find a solution for that. It seems me that we have only option to make some templates like top/bottom/top-right/top-left/bottom-right/bottom-left and use it to pass as props for styled component and implement logic inside styled component.

One day this kind implementation was made for special design request.